tests/ext/bottle/test_bottle.py

```diff
@@ -0,0 +1,302 @@
+import pytest
+from bottle import Bottle, request, response, template, view, HTTPError, TEMPLATE_PATH
+from webtest import TestApp as WebApp
+
+from aws_xray_sdk import global_sdk_config
+from aws_xray_sdk.ext.bottle.plugin import XRayPlugin
+from aws_xray_sdk.core.context import Context
+from aws_xray_sdk.core import lambda_launcher
+from aws_xray_sdk.core.models import http, facade_segment, segment as segment_model
+from tests.util import get_new_stubbed_recorder
+import os
+
+
+# define Bottle app for testing purpose
+TEMPLATE_PATH.insert(0, os.path.dirname(__file__) + '/views')
+app = Bottle()
+
+@app.route('/ok')
+def ok():
+    response_data = 'ok'
+    # Bottle not always set Content-Length header
+    response.content_length = len(response_data)
+    return response_data
+
```

Comment:
Can we modify spacing in this file ? One line space between each functions

Reply:
sure, simply followed https://github.com/aws/aws-xray-sdk-python/blob/master/tests/ext/flask/test_flask.py style

Reply:
I would change it if you insist, but it conforms PEP8 and is consistent with other tests in a project. 
Again `Black` could eliminate checking formatting in code review.

Reply:
okay never mind. That's fine.